data/pidgin/cpsAssets/23248703.json

```diff
@@ -71,7 +71,7 @@
               {
                 "headlines": {
                   "headline": "Lorem ipsum dolor sit amet, consectetur",
-                  "overtyped": "<Media Asset Page>",
+                  "overtyped": "Media Asset Page",
```

Comment:
Any change we could make the link names slightly different for each link? (Or does that have to be done in CPS?) e.g. Media Asset Page 1, Media Asset Page 2 etc

Reply:
This fixture should ideally map to what is in CPS, but I have updated the fixture we actually use for testing the internal links to be Media Asset Page 1, 2, 3 etc

Can't update this fixture fully as it is what we use for testing CpsMain, and resyncing it with what is in CPS breaks most of those tests.

Out of scope for this PR, but will create an issue to update this Pidgin fixture, and associated unit tests over in the CPS page